packages/math-input/src/components/keypad-container.tsx

```diff
@@ -26,7 +26,7 @@ const {row, centered, fullWidth} = Styles;
 
 type Props = {
     active?: boolean;
-    extraKeys?: Array<Key>;
+    extraKeys?: Array<Key>; // Not used?
```

Comment:
It appears to be passed through to the the various keypads in `renderKeypad`?  Or is it not used inside those keypads?